questions.md

```diff
@@ -461,5 +461,9 @@ in your own life or been asked by someone?
 
 Please share!
 
+<a href="https://twitter.com/home?status=https%3A//github.com/dwyl/start-here/blob/master/questions.md">Share on Twitter</a><br>
+<a href="https://www.facebook.com/sharer/sharer.php?u=https%3A//github.com/dwyl/start-here/blob/master/questions.md">Share on Facebook</a><br>
+<a href="https://plus.google.com/share?url=https%3A//github.com/dwyl/start-here/blob/master/questions.md">Share on Google+</a><br>
```

Comment:
Google Plus will be "sunset" very soon: https://plus.google.com/+googleplus/posts/NrUSwKLr7mU
![image](https://user-images.githubusercontent.com/194400/52519283-cd025a00-2c50-11e9-8c48-1ae04c46df92.png)

Probably worth _excluding_ this link as people are only pouring their efforts into a _zombie_ platform.